baseband/vdif/payload.py

```diff
@@ -43,6 +69,7 @@ def init_luts():
 
 
 def decode_2bit(words):
+    """Decodes int32 words into 2-bit samples."""
```

Comment:
The default words are actually `uint32`, though really it doesn't matter as it is converted to bytes anyway.
Maybe
```
Decoder for data stored using 2 bits per sample.
```